pi_processing/processors.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+function error_section_value_error_between(
```

Comment:
Some duplication between these functions which would be nice to somehow get rid of; at least it should be possible to move the import to the caller. Note that there's also an `appendXML` method you can use on DOMElement.